### PR TITLE
Fix rounding error in cart

### PR DESCRIPTION
**Pull request by tutturen**
* Fixed bug in `getPrice` function
* Added tests

### CODE REVIEW

**Comment by tutturen** [comment]
Closes #41

File: package.json [hunk 1]
```diff
@@ -3,13 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "apollo-cache-inmemory": "^1.1.11",
+    "apollo-cache-persist": "^0.1.1",
+    "apollo-client": "^2.2.7",
+    "apollo-link": "^1.2.1",
+    "apollo-link-context": "^1.0.7",
+    "apollo-link-http": "^1.5.3",
+    "apollo-link-state": "^0.4.1",
```
Comment (tobinus) [inline package.json]: Looks like this should wait until #25 is merged..?

Comment (tobinus) [reply package.json]: It's a bit difficult to review this before #25 is merged, because this is now equal to #25 plus a bit more.

**Review by tobinus — APPROVED** [review]
I liked that you made tests for this, made it way easier to try out other approaches.

**Pull request merged**